e2e/android/build.gradle

```diff
@@ -36,41 +31,11 @@ allprojects {
         }
         maven {
             // Android JSC is installed from npm
-            url(new File(['node', '--print', "require.resolve('jsc-android/package.json')"].execute(null, rootDir).text.trim(), '../dist'))
+            url(new File(['node', '--print', "require.resolve('jsc-android/package.json', { paths: [require.resolve('react-native/package.json')] })"].execute(null, rootDir).text.trim(), '../dist'))
         }
 
         google()
         mavenCentral()
         maven { url 'https://www.jitpack.io' }
     }
-    configureSpotless(it)
-}
-
-def configureSpotless(Project project) {
```

Comment:
Spotless seems to no longer be compatible with react-native 0.73. This was used for linting, but the android project is autogenerated anyways, so I don't see much value in trying to make linting work again. So, I decided to remove spotless altogether from the sample android app.